screens/produtlist/product.css

```diff
@@ -0,0 +1,238 @@
+    --box-shadow:rgba(19, 1, 1, 0.822);
+    --border-color:hsl(39, 77%, 83%);
+  
+    
+    
+  }
+  
+  
+  body {
+      font-family: sans-serif;
+      margin: 0px;
+      padding: 0px;
+     
+    }
+    html{
+      font-size: 62.5%;
+    }
+    
+    .containr {
+      display: grid;
+      grid-template-columns: 0.5fr 1fr 1fr;
+      grid-template-rows: 0.2fr 1.2fr 1.5fr;
+      grid-template-areas:
+        "nav nav nav nav"
+        "sidebar main main main"
+        "sidebar main main main"
+        "...... main main main";
+    
+      justify-content: center;
+      background-color: var(--primary-font-colour);
+      position: relative;
+    }
+    .grid {
+      display: grid;
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+      flex-wrap: wrap;
+      margin: 3rem;
+      justify-content: space-between;
+      grid-gap:4rem 4rem;
+      width: 90%;
+    }
+    
+    nav {
+      background-color: var(--cta-color);
+      grid-area: nav;
+      height: 5rem;
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      width: 100%;
+      position: fixed;
+      color: var(--primary-font-colour);
+      font-size: 1.5rem;
+    }
+    .card-prododuct-result{
+      position: absolute;
+      top: 4.5rem;
+      font-size: 2rem;
+    
+      /* text-align: center; */
+      margin-left: 0px;
+    }
+    .logo{
+      font-size: 2rem;
+    }
+    .search-data {
+      height: 2rem;
+      width: 20%;
+      padding: 8px 10px;
+      border-radius: 8px;
+      border: 2px solid var(--cta-color);
+    }
+    .fa-search {
+      position: absolute;
+      top: 20px;
+      margin-left: 12rem;
+      color: var(--cta-color);
+    }
+    .fa-heart-half {
+      color: var(--primary-font-colour);
+    }
+    
+    .cart-icon {
+      position: absolute;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      top: -4px;
+      padding: 5px;
+      height: 1rem;
+      width: 1rem;
+      background-color: var(--error-color);
+      border-radius: 50%;
+    }
+    li {
+      display: inline-block;
+      margin: 2rem;
+     
+    }
+    main {
+      background-color: var(--primary-font-colour);
+      grid-area: main;
+      padding: 10px;
+    
+    }
+    
+    img {
+      width: 100%;
+      object-fit: cover;
+      height: 20rem;
+    }
+    .card-image-one {
+      width: 20rem;
+      height: 38rem;
+      background-color: var(--primary-font-colour);
+      border: 1px solid var(--cta-color);
+      border-radius: 8px;
+      display: flex;
+      flex-direction: column;
+      padding: 5px;
+      align-self: center;
+    }
+    .card-image-data {
+      text-align: center;
+      font-size: 1.5rem;
+    }
+    .btn {
+      font-size: 1.5rem;
+      border-radius: 10px;
+      border: none;
+      padding: 5px;
+      text-align: center;
+      cursor: pointer;
+    
+      font-weight: 500;
+    }
+    .btn-icon-add {
+      background-color: gray;
```

Comment:
you could create a variable for color `gray` like `--gray-color`

Reply:
updated